### PR TITLE
JavaDoc

### PR DESCRIPTION
**Pull request by tylerdvanl**
Added javadoc headers and comments, leaving out constructors and toStrings

### CODE REVIEW

File: src/cecs277TermProject/Logger.java [hunk 1]
```diff
@@ -55,19 +67,35 @@ public void log(String message)
 		out.println(String.format("[%1$s] %2$s", timestamp, message));
 	}
 
+
+	/**
+	 * Gets the filename of the output file.
+	 * @return file name of the output file.
+	 */
+
```
Comment (thaolydo) [inline src/cecs277TermProject/Logger.java]: nit: extra line

File: src/cecs277TermProject/Logger.java [hunk 2]
```diff
@@ -55,19 +67,35 @@ public void log(String message)
...
 	{
 		return this.filename;
 	}
 
+
+	/**
+	 * Gets the file object of the output file itself.
+	 * @return The output file itself.
+	 */
+
```
Comment (thaolydo) [inline src/cecs277TermProject/Logger.java]: nit: extra line

File: src/cecs277TermProject/Logger.java [hunk 3]
```diff
@@ -55,19 +67,35 @@ public void log(String message)
...
 	public File getFile()
 	{
 		return this.outFile;
 	}
 
+
+	/**
+	 * Closes the logger's printwriter, effectively turning this class off.
+	 */
+
```
Comment (thaolydo) [inline src/cecs277TermProject/Logger.java:93]: same here

**Review by thaolydo — APPROVED** [review]
Please fix a few minor nit issues and feel free to merge after that.

**Comment by tylerdvanl** [comment]
I fix.

**Review by thaolydo — APPROVED** [review]

**Pull request merged**